answers.md

```diff
@@ -0,0 +1,80 @@
+# mssw Chapter 12 Answers
+
+## 1. Enumeration constants are said to be *self-typed*. What does this mean?
+
+I didn't really understand this but i think they mean that the type of the enum is the class name that it is contained in.
```

Comment:
Yeah, it more or less just means _what type do the enum constants have, given that they're not explicitly typed_.